### ISSUE TITLE
NTR: subdistal appendages

### ISSUE THREAD

**Issue by krchristie** [geneontology/go-ontology#14285]
A reviewer of the cilia GO ontology manuscript has requested an additional component term as part of the eukaryotic cilium. @paolaroncaglia, @JohnvanDam, and I need to work on this promptly to meet October 30 deadline to submit revisions.

> 2: similarly, the eukaryotic cilium is also only missing one component: subdistal appendages. 
> There are several proteins localizing there are described e.g. in  10.1242/bio.20122337 and 
> https://doi.org/10.1016/j.devcel.2016.10.006.

link for 10.1242/bio.20122337: http://bio.biologists.org/content/1/10/965 
```
Sonnen KF, Schermelleh L, Leonhardt H, Nigg EA. 3D-structured illumination
microscopy provides novel insight into architecture of human centrosomes. Biol
Open. 2012 Oct 15;1(10):965-76. PMID: 23213374
```

https://doi.org/10.1016/j.devcel.2016.10.006. =
```
Mazo G, Soplop N, Wang WJ, Uryu K, Tsou MB. Spatial Control of Primary
Ciliogenesis by Subdistal Appendages Alters Sensation-Associated Properties of
Cilia. Dev Cell. 2016 Nov 21;39(4):424-437. PMID: 27818179
```

which is also referred to in this interesting commentary:
```
Galati DF, Mitchell BJ, Pearson CG. Subdistal Appendages Stabilize the Ups and
Downs of Ciliary Life. Dev Cell. 2016 Nov 21;39(4):387-389. PMID: 27875681.
```

**Comment by paolaroncaglia** [comment]
Copying from an email from @JohnvanDam for internal reference:
"I have to check if the sub distal appendages should maybe be a synonym to an already existing term. I'll check that this wednesday, when I get back at work."

**Comment by paolaroncaglia** [comment]
Quick notes:

From the first paper (http://bio.biologists.org/content/1/10/965):
“Of the two centrioles, the older one (frequently referred to as the mother) carries distinctive distal and subdistal appendages (Paintrand et al., 1992), which confer important functional asymmetry (Bornens, 2012; Nigg and Stearns, 2011).”
So at a first glance it sounds like we may want to create ‘subdistal appendage’ as part_of GO:0005814 ‘centriole’. (At present, ‘centriole’ has no children.) However, ‘appendage’ is ambiguous (see ‘GO:0035107 ‘appendage morphogenesis’) and I’d rather use it as a synonym. Or call it explicitly ‘centriole subdistal appendage’ - we have previously allowed this type of specification in GO where necessary to disambiguate (see some dinoflagellate terms). Also, the reviewer asks for ‘subdistal appendages’, but we may need to think about ‘distal’ too if appropriate - as a term or synonym.  

The second paper (http://www.sciencedirect.com/science/article/pii/S1534580716307195) suggests we should add sDAP as a synonym: 
“…centriole subdistal appendages (sDAP)…”. 

We should also annotate the relevant human proteins, see section on “Appendage proteins” in http://bio.biologists.org/content/1/10/965. Proteins are also described in the second paper.

**Comment by krchristie** [comment]
The term 'ciliary transition fiber' already has several 'distal appendage' synonyms. The question I would ask is whether the subdistal appendages are also transition fibers, in which case, we might need to have separate terms for both distal and subdistal appendages, each of which could be a type of transition fiber. I agree that terms called 'distal appendage' or 'subdistal appendage' would need to be explicitly called 'centriole ...' 

```
[Term]
id: GO:0097539
name: ciliary transition fiber
namespace: cellular_component
def: "A nine-bladed, propeller-like protein complex that links the distal end of the basal body and the cilium to the plasma membrane. Functions in protein sorting and gating (i.e. active and passive transport of proteins in and out of the cilium)." [GOC:cilia, GOC:kmv, GOC:krc, PMID:22653444, PMID:24231678, PMID:5064817, PMID:5335827]
comment: In mammals, ciliary transition fibers comprise at least five components (Ccdc41/Cep83, Cep89/Cep123, Sclt1, Fbf1, and Cep164) (PMID:24469809).
synonym: "centriolar distal appendage" RELATED [PMID:24469809]
synonym: "cilial transition fiber" EXACT []
synonym: "cilial transition fibre" EXACT []
synonym: "ciliary transition fibre" EXACT []
synonym: "cilium transition fiber" EXACT []
synonym: "cilium transition fibre" EXACT []
synonym: "distal appendage of basal body" RELATED [GOC:krc]
synonym: "distal appendage of centriole" RELATED [GOC:krc]
synonym: "distal appendage of mother centriole" RELATED [GOC:krc]
synonym: "transition fiber" BROAD []
synonym: "transition fibre" BROAD []
is_a: GO:0043234 ! protein complex
is_a: GO:0044441 ! ciliary part
created_by: paola
creation_date: 2013-12-03T16:46:37Z

```

**Comment by krchristie** [comment]
This paper:

**Garcia-Gonzalo FR, Reiter JF**. Open Sesame: How Transition Fibers and the
Transition Zone Control Ciliary Composition. Cold Spring Harb Perspect Biol. 2017
Feb 1;9(2). Review. PubMed PMID:27770015

confirms our existing equivalence of distal appendages' with 'transition fibers' 

> Basal bodies start their lives as mother centrioles, distinguished from daughter centrioles by the distal appendages through which they dock to the vesicles that are eventually remodeled into the ciliary membrane (Sorokin 1962; Lu et al. 2015). When associated with the ciliary membrane or its precursors, centriolar distal appendages are referred to as transition fibers (TFs) or alar sheets.

and does not suggest that subdistal appendages should also be considered as 'transition fibers'

**Comment by krchristie** [comment]
from Mazo G et al. 2016 (PMID: 27818179):

> DAP and sDAP are distinct structures composed of different core components:
- **DAP**: CEP83, CEP89, SCLT1, CEP164, and FBF1
- **sDAP**: ODF2, CNTRL, and NIN  

from Galati et al. 2016 (PMID: 27875681)

> Centrioles acquire subdistal appendages (sDAPs) during primary cilium formation.

From what I've looked at, I do not think that any of our existing terms represent distal appendages and agree with the reviewer that we need a new term for these.

Since they form during ciliogenesis, they could be is_a 'ciliary part'.

**Comment by JohnvanDam** [comment]
Interesting how I seem to have missed sDAPs completely... I agree that DAP and sDAP appear to be two different structures. To me it seems like DAP is the same as transition fibers though, but from Karen's previous post I guess literature is conflicting on this. Figure 3 in https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4113101/ has some nice detailed EM photographs showing the difference between DAP and sDAP. Comparing the protein annotations for transition fiber to the list above it seems that some of the protein annotations need to be reassigned after the sDAP term is created.

In addition, distal appendage seems to be used more often than transitional fiber, when searching pubmed. We might consider making distal appendage the canonical name if we agree that DAP and transitional fibers constitute the same structure.

**Comment by krchristie** [comment]
Sorry if it was confusing that I was just wondering if sDAPs are also transition fibers. However, some reading I did yesterday (Garcia-Gonzalo et al. 2017, PMID:27770015) confirms that DAP is the same as transition fiber (and sDAP is not), so I am in agreement with you there. 

I have also been wondering if we should change the name from "transition fiber" to "distal appendage". If PubMed is indicating that "distal appendage" is more commonly used, then I agree it might be good to make it the primary term name and convert "transition fiber" to a synonym.

I also agree with Paola's suggestion to add DAP and sDAP as synonyms.

**Comment by paolaroncaglia** [comment]
Hi @krchristie and @JohnvanDam,

Thank you both for your feedback on this. You have both read a lot more than I have on those cellular components, so I’ll refer to your judgment. My only comment would be that when we created the cilia terms, ‘transition fiber’ sounded like well-accepted nomenclature to me, so I might be a bit reluctant about swapping it for a synonym, but as I said I’m not the major expert here. Perhaps, if there are still any grey areas we’d like validation on, we could ask for a third opinion? Who could be a friendly expert you’ve already been in touch with, and who’d be available to provide feedback promptly? So we can go ahead with adding/editing terms and annotations. Thanks!

**Comment by JohnvanDam** [comment]
I guess renaming transition fibers would not be important as it is captured via synonym anyway, and as you say the current name is well accepted in literature. As for experts to contact, I am now browsing through pubmed for names that may be considered expert on centrioles and distal appendages: JF Reiter (UCSF) comes up regularly in the context of both centrioles and basal bodies. Although I did not have previous contact before, he seems a nice enough fellow. I would not know who in my Syscilia network would be specifically knowledgeable about sub-distal vs distal appendages, but we could maybe ask Colin Johnson (Leeds University) if he knows, or at least knows someone. Shall I send him an email?

**Comment by paolaroncaglia** [comment]
Thanks @JohnvanDam,
I wouldn't mind you asking Drs. Reiter or Johnson, unless @krchristie has any objection.

**Comment by krchristie** [comment]
Transition fiber is well accepted nomenclature, but distal appendages are also well accepted nomenclature. The biggest issue I've had in annotating these seems to be that many papers use one name or the other without mentioning the other name, but as long as we have both names attached to the same term, curators and users will be able to find it. GO often goes with the most commonly used name as the primary name.

I've met Jeremy Reiter at a cilia meeting, though haven't had any further contact with him subsequently. Rereading the quote from a review from his lab that I included earlier in this thread also makes me wonder if there is a subtle distinction between distal appendages and transition fibers based on the fact that this quote specifies the "centriolar distal appendages are referred to as transition fibers" "**When** associated with the ciliary membrane or its precursors". This might suggest a need for terms for both "centriolar distal appendages" and "transition fibers" separately from each other. I would be quite cautious about this course as it seems it would be very difficult to annotate these accurately.

I don't object to contacting people for their input, though we should be aware that adding non-GOC people to the conversation often makes it take longer to get to resolution. But if John wants to shepherd that conversation, I don't object.

**Comment by krchristie** [comment]
While trying to come up with a proposal for the new "centriolar subdistal appendage" term, I've seen some additional comments that also suggest that at least sometimes there is a distinction in the use of "distal appendage" to "transition fiber" that corresponds to a maturation of the structure from the DAPs forming on the mother centriole to becoming transition fibers when they form the connections of a fully assembled cilium, e.g. this one from this paper cited by the existing "transition fiber" term:

**Wei Q et al. Transition fibre protein FBF1 is required for the ciliary entry of assembled
intraflagellar transport complexes. Nat Commun. 2013;4:2750. PMID: 24231678.**

> transition fibres (TFs; transformed from the distal appendages of the mother centriole) 

However, from an annotator's perspective, I don't think it will be useful, or even possible, to make a meaningful distinction between a "distal appendage" and a "transition fiber" since I think many papers still refer to these structures as distal appendages even when looking at fully assembled cilia. I'd be fine with just leaving this term as is, though I don't object if John wants to contact researchers.

**Comment by krchristie** [comment]
We also can proceed with adding the requested term for centriolar subdistal appendages without resolving the issue regarding the name of the term for "centriolar distal appendages" or "transition fibers", so here is a proposal for a new term:

name: centriolar subdistal appendage
definition: A structure containing ODF2, CNTRL, NIN, CCDC120c and CCDC68 which forms on the mother centriole, adjacent and proximal to a centriolar distal appendage, during cilium formation.
def dbxrefs: Mazo G et al. 2016 (PMID: 27818179); GOC:cilia, PMID:28422092 
synonym: sDAP (RELATED) 

and the same parentage as the current "transition fiber" term, e.g.:
subclass_of: cilium part
subclass_of: protein complex

I'm hesitant to try to say too much about what the complex does as these two papers cited seem to be among the among the first to try to characterize what the subdistal appendages do, so it may be premature to try to say much.

**Comment by JohnvanDam** [comment]
The biggest issue regarding these terms is that they are called different depending on the situation. Which is fair, but I think essentially these are the same structures. I think the quote about TF's being "transformed from the distal appendages of the mother centriole" is a way for the authors to deal with the two nomenclatures, as so far as I can tell the only known difference is whether it attaches to the plasma membrane or not. I would keep transition fiber and distal appendage as synonyms, and maybe add a comment that it could be possible that the composition of this structure might change depending on the state of the centriole/basal body.

I will not contact Colin of JF Reiter at this point, to avoid going of track with what we need to do at this point. Maybe we can revisit once it becomes relevant for annotation when more conclusive research emerges about possible differences between distal appendages and transition fibers.

The proposed term definition by Karen for subdistal appendage looks good to me.

**Comment by paolaroncaglia** [comment]
@krchristie @JohnvanDam

I agree that if the literature is in its ‘early days’ wrt sub distal appendages, we may not want to say too much. Karen’s latest proposal above would be good with me, except for a minor point in the definition, i.e. addressing the need to try and make the definition as species-generic as possible. So perhaps
“A protein complex which assembles on the mother centriole during cilium formation, adjacent and proximal to a centriolar distal appendage. In human [it is human, right?], it contains ODF2, CNTRL, NIN, CCDC120c and CCDC68.”
I would also add PMID:23213374 among def dbxrefs (“3D-structured illumination microscopy provides novel insight into architecture of human centrosomes”), as that was suggested by reviewer 1.

As for expanding the definition comment to ‘ciliary transition fiber’ along the lines of “the composition of this structure might change depending on the state of the centriole/basal body”, I do not disagree and will leave it to you two, if you’d like, to choose the most appropriate wording.

Lastly, who should annotate those proteins? @krchristie, can you do that or should we ask UniProt?

Thanks!

**Comment by krchristie** [comment]
Thanks for the suggestion on the definition @paolaroncaglia. I will definitely include that it is human. I don't know why I didn't include that. I normally do... I'll add the term today so it is available in Protein2GO as soon as possible.

I think it might be simplest if I just annotate them myself. It's only two papers.

**Comment by krchristie** [comment]
I've added the new term for 'centriolar subdistal appendage' as I proposed with Paola's suggestions incorporated, and also with some synonyms following the pattern of the existing synonyms for the 'ciliary transition fiber' term, viewable here: 
https://github.com/geneontology/go-ontology/pull/14339/files

Since I forgot to close this automatically, I think I'll just leave it open to remind me to do the annotations when the term is available in P2GO.

**Comment by paolaroncaglia** [comment]
@krchristie Thanks for adding the term - and the annotations later on!

**Comment by paolaroncaglia** [comment]
Hi @krchristie,

Is the new term 'centriolar subdistal appendage' now available in P2GO? If so, and compatibly with your other commitments, could you please add a comment on this ticket when the annotations are ready, so we can add their details in the re-submission letter. Many thanks!

**Comment by paolaroncaglia** [comment]
@krchristie
P.S. and indicate the expected release date of those human annotations, thanks.

**Comment by krchristie** [comment]
It doesn't look like the new GO term has made it into P2GO yet. I had tried yesterday, and just checked again now.

Regarding the expected release date, I have no idea. Maybe it would be easier for you,  @paolaroncaglia, to find out from your EBI colleagues when the next planned release date is.

**Comment by krchristie** [comment]
@paolaroncaglia - I have made a number of annotations for proteins that are part of the subdistal appendages from two papers. I've downloaded the annotations as GPAD files directly from P2GO, so I'll email you those.

I think that complete annotation of PMID:23213374 would require a one or two more terms, highlighted in this quote from this paper:

Together, our studies identified redundant functions of sDAP and the proximal end factor C-Nap1 in centrosome cohesion, **cilia-to-Golgi association**, and **ciliary pit maintenance**, critical for the spatial control of ciliogenesis. I'll put this new stuff in a different ticket...

These two papers might also suggest a need for more granular terms for the centriole. Currently both "mother centriole" and "daughter centriole" are both synonyms for centriole, but there are things that localize differentially to one or the other. Then these papers also make a point about localization to the proximal end of the centriole, but then we get into a proliferation of combinations of proximal vs distal with mother vs daughter... so I think I'll just leave that be.
